src/components/Dialog/ModelessDialog.tsx

```diff
@@ -95,6 +95,7 @@ export const ModelessDialog: React.VFC<Props & BaseElementProps> = ({
 }) => {
   const portalContainer = useRef(document.createElement('div')).current
   const wrapperRef = useRef<HTMLDivElement>(null)
+  const [wrapperPosition, setWrapperPosition] = useState<DOMRect | undefined>(undefined)
```

Comment:
nits: 意図がなければ undefined を明示する必要はなさそう（初期状態で `DOMRect | undefined` になる

```suggestion
  const [wrapperPosition, setWrapperPosition] = useState<DOMRect>()
```

Reply:
了解!
手元ではなってそうだった。なんでだろう
![image](https://user-images.githubusercontent.com/19403400/138215763-11930928-29cc-4a50-8f1f-e97c0dcd9815.png)

Reply:
ああ、`()` だけだった。理解。アザス、できそう。
だけど、 `undefined` って明示したいかも〜。